### PR TITLE
Fix (UI): Fix lighthouse accessibility errors

### PR DESCRIPTION
**Pull request by SoldierBoy31**
## Description

This fixes accessibility errors described on jenkins CI/CD (from May 27, 2022).

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [X] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>

Launch lighthouse accessibility report on pages: Login, Resources status and Authentification.

#### Community contributors & Centreon team

- [X] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by bdauria — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -88,10 +88,11 @@
     "@mui/lab": "^5.0.0-alpha.63",
     "@mui/material": "^5.4.0",
     "@mui/styles": "^5.3.0",
+    "@mui/x-date-pickers": "^5.0.0",
     "@tanstack/react-query": "^4.1.0",
     "@visx/visx": "2.9.0",
     "axios": "^0.25.0",
-    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
+    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#fix-contrast-ratio",
```
Comment (JKancel) [inline package.json]: I assume that it should point to develop after merge :
```suggestion
    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
```

File: tests/e2e/cypress/integration/Resources-status/02-actions/index.ts [hunk 2]
```diff
@@ -18,15 +18,14 @@ before(() => {
 
 When('I select the acknowledge action on a problematic Resource', () => {
   cy.contains(serviceInAcknowledgementName)
-    .parents('div[role="row"]:first')
+    .parent()
+    .parent()
+    .parent()
+    .parent()
```
Comment (jdelpierre) [inline tests/e2e/cypress/integration/Resources-status/02-actions/index.ts:24]: it's not possible to use a particular selector ?

https://docs.cypress.io/api/commands/parents#Syntax
https://docs.cypress.io/api/commands/parentsuntil#Syntax

File: tests/e2e/cypress/integration/Resources-status/02-actions/index.ts [hunk 3]
```diff
@@ -50,15 +49,14 @@ Then('the problematic Resource is displayed as acknowledged', () => {
 
 When('I select the downtime action on a problematic Resource', () => {
   cy.contains(serviceInDowntimeName)
-    .parents('div[role="row"]:first')
+    .parent()
+    .parent()
+    .parent()
+    .parent()
```
Comment (jdelpierre) [inline tests/e2e/cypress/integration/Resources-status/02-actions/index.ts:55]: same comment

File: www/front_src/src/components/Wallpaper/BackgroundImage.tsx [hunk 4]
```diff
@@ -34,7 +32,7 @@ const BackgroundImage = ({ image }: Props): JSX.Element => {
 
   return (
     <Fade in={not(isNil(image))}>
-      <div aria-label={labelCentreonWallpaper} className={classes.wallpaper} />
+      <div className={classes.wallpaper} id="Centreon_wallpaper" />
```
Comment (jdelpierre) [inline www/front_src/src/components/Wallpaper/BackgroundImage.tsx]: @Thebarda What do you think about this syntax ?
```suggestion
      <div className={classes.wallpaper} id="centreon-wallpaper" />
```

File: package.json (hunk 1 above)
Comment (Thebarda) [reply package.json]: Yup. I did it in order to check the Lighthouse report generated by the CI

File: tests/e2e/cypress/integration/Resources-status/02-actions/index.ts (hunk 2 above)
Comment (Thebarda) [reply tests/e2e/cypress/integration/Resources-status/02-actions/index.ts:24]: No

Comment (jdelpierre) [reply tests/e2e/cypress/integration/Resources-status/02-actions/index.ts:24]: ok

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**